app/views/help/helpcenter.html.erb

```diff
@@ -1,38 +1,37 @@
-          
-           <!-- To set up the Bootstrap grid /add comment for what this will be for later -->
-          <div class="container">
-            <div class="row">
-              <div class="col">
-                <h3>Student Questions</h3>
-              </div>
-                <div class="col">
-                </div>
-                <div class="col">
-                </div>
-                <div class="col">
-               <h3>Mentor Questions</h3> 
-              </div>
-            </div> <!--  /where it ends  --> 
-          </div> <!--   where the /.container ends -->
+  <h1 class="display-4 mt-5">Carrot University Help Center</h1>
+  <p class="lead mb-f">Frequently Asked Questions From Students and Mentors</p>
 
+  <div class="col-lg-6 mx-auto">
+    <div class="input-group mb-3">
+      <input type="text" class="form-control" placeholder="Search for ..." aria-label="Search for ..." aria-describedby="button-addon2">
+      <div class="input-group-append">
+        <button class="btn btn-success" type="button">Go!</button>
+      </div>
+    </div>
 
-
-        <!-- Optional JavaScript -->
-        <!-- jQuery first, then Popper.js, then Bootstrap JS -->
-        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
-        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
-        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    <!-- To set up the Bootstrap grid /add comment for what this will be for later -->
+    <div class="container">
+      <div class="row">
+        <div class="col">
+          <h3>Student Questions</h3>
+        </div>
+        <div class="col">
+        </div>
+        <div class="col">
+        </div>
+        <div class="col">
+          <h3>Mentor Questions</h3>
+        </div>
+      </div> <!--  /where it ends  -->
+    </div> <!--   where the /.container ends -->
 
 
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
```

Comment:
I'm wondering how valuable it is to actually have these in the gemfile instead of using the CDN. Would be a good opportunity to start exploring how rails handles javascript. Again, not necessary, just something to consider if time and opportunity allow.